packaging/artifact-registry-yum-plugin.el7.spec

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Name: yum-plugin-artifact-registry
+Epoch:   1
+Version: %{_version}
+Release: g1%{?dist}
+Summary: Yum plugin for Google Artifact Registry
```

Comment:
I think the official product name is Artifact Registry (rather than Google Artifact Registry).